modules/efk-logging-fluentd-undefined.adoc

```diff
@@ -5,22 +5,18 @@
 [id="efk-logging-fluentd-undefined_{context}"]
 = Configuring how the log collector normalizes logs 
 
-Cluster Logging uses the link:https://github.com/ViaQ[ViaQ common data model] to deal with problem that arise from conflicts 
-and inconsistencies in log data collected from different subsystems. {product title} allows you to configure  
+Cluster Logging uses the link:https://github.com/ViaQ/fluent-plugin-viaq_data_model[ViaQ common data model] to deal with problem that arise from conflicts 
```

Comment:
Fluentd uses that plugin.  Rsyslog uses several different ways.

Reply:
@richm Is there a good link to use for info on ViaQ? That is what is needed here. What is the ViaQ data model?